src/annotated_lcf.sig

```diff
@@ -1,5 +1,11 @@
+(* In order to support a more robust form a failure
+ * for a tactic, we have a notion of an LCF system
+ * which knows how to annotate tactics. This lets a
+ * failing tactic give information about what went wrong.
+ *)
```

Comment:
Well, in the current version of "annotated LCF", the failing tactic doesn't really give any more information about what went wrong then it did before, but it does give the system the ability to embed other information (whose use and scope is not known from the point of view of this signature).